dockerfiles/gomoderator/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM golang:alpine as builder
+
+RUN apk --update --no-cache add make git g++ linux-headers python3
+
+ARG VDB_VERSION=staging
+ENV GO111MODULE on
+
+WORKDIR /go/src/github.com/makerdao
+RUN git clone https://github.com/makerdao/vulcanizedb.git
+RUN git clone https://github.com/makerdao/vdb-mcd-transformers.git
+WORKDIR /go/src/github.com/makerdao/vulcanizedb
+RUN git checkout $VDB_VERSION
+RUN go build
+WORKDIR /go/src/github.com/makerdao/vdb-mcd-transformers
+RUN git checkout $VDB_VERSION
+RUN go build
+
+# app container
+FROM golang:alpine
+WORKDIR /go/src/github.com/makerdao/vulcanizedb
+
+# add certificates for node requests via https
+# bash for wait-for-it
+RUN apk update \
+        && apk upgrade \
+        && apk add --no-cache \
+        ca-certificates \
+        bash \
```

Comment:
Super small, but i'm not sure if we need these two lines. I don't think we're using bash in this dockerfile (we're using it for the wait-for-it script for some other commands). And then based on the comment about the certificates, I _think_ that's just for if we're making node requests. Not 100% sure on that though.

Reply:
Cool yeah, I think we can get rid of all of it.